File: unique.js
let nums = [1, 2, 3, 3, 3, 3, 4, 5];
let nums2 = [];

for (let i = 0; i <= nums.length - 1; i++) {
  if (nums[i] != nums[i + 1]) {
    nums2.push(nums[i]);
  }
}

console.log(nums2);

File: unique.py
nums = [1, 2, 3, 3, 3, 3, 4, 5]
nums2 = []

for i in range(len(nums) - 1):
    if nums[i] != nums[i + 1]:
        nums2.append(nums[i])

nums2.append(nums[-1])

print(nums2)
